### ISSUE TITLE
There are differences between SSIM LOSS and MEF SSIM LOSS

### ISSUE THREAD

**Issue by JohnTargaryen** [exceptionLi/Imagefusion_deepfuse#1]
hey there, I notice that your code for loss function in ssim_loss_function.py is quite similar to the code I found in https://stackoverflow.com/questions/39051451/ssim-ms-ssim-for-tensorflow

However, this seems to be **SSIM LOSS** function instead of **MEF SSIM LOSS** function. In  
_K. Ma, K. Zeng, and Z. Wang. Perceptual quality assessment for multi-exposure image fusion. IEEE Transactions on Image Processing, 24(11):3345–3356, 2015._ , the author mentioned that "Direct use of the SSIM algorithm [27], however, is impossible, which requires a single perfect quality reference image. "

The paper _Deepfuse_'s work is exactly on MEF(multi-exposure image fusion), so I would think there should be differences between SSIM LOSS and MEF SSIM LOSS, in other words, the loss function in your code may be incorrect?

Look forward to your reply, thx

**Comment by hli1221** [comment]
Thank you very much for point out these mistakes. 
Yes, you are right. The loss function is not the same with DeepFuse. And I have noticed these before.
After I finish my project, I will try to fix these mistakes.

I will add a statement in 'README.m' file.

**Comment by JohnTargaryen** [comment]
> Thank you very much for point out these mistakes.
> Yes, you are right. The loss function is not the same with DeepFuse. And I have noticed these before.
> After I finish my project, I will try to fix these mistakes.
> 
> I will add a statement in 'README.m' file.

You're welcome. Thank you for sharing your work.
